### ISSUE TITLE
Wireframe: News article (UP)

### ISSUE THREAD

**Issue by jillian-hammer** [CDCgov/prime-reportstream#11094]
## User story

As a ReportStream prospective/current user, I want the website content to be up-to-date, accurate and timely so I can  trust the information to inform my decisions in my relationship with ReportStream. 

## Background & context

This ticket captures the work of putting together the **News article (UP)** using the detail page template for dev ready handoff.

## Working links

- Figma: [Working file](https://www.figma.com/file/Du7feM8huRDAAZCQcUIj93/Website-Redesign?type=design&node-id=225%3A822&mode=design&t=oaE1uhsUhwsV6zy0-1)
-  Content doc: [News article](https://docs.google.com/document/d/1ANLRiKWv9N32bJZSHrg9-WQu4FxVe1JsXzZZJMv83TY/edit?usp=sharing)

## Acceptance criteria
**Design**
-   [x] Design includes latest content updates
-   [x] Capture questions that come up during the content and design work in the [Website redesign research plan](https://docs.google.com/document/d/1Hmxu_mTGvSJ0RnfS_SXoY84dBeAyUuy7KsCL08N0azg/edit#heading=h.sqsn2oxgx0ou)

**Review**
-   [x] Reviewed with design and/or design lead
-   [x] Reviewed with content strategist for final approval
-   [x] Reviewed with product and engineering

Follow up ticket needed: To update content

**Comment by yudonoodles** [comment]
Reviews are complete! Closing it since the design is done. Will need to create a new ticket to capture updating content when that's ready

cc: @audreykwr @chris-kuryak @jillian-hammer 

Moved the completed layout/design to page on Figma as we wait on content to be finalized ["On hold: News"](https://www.figma.com/file/Du7feM8huRDAAZCQcUIj93/Website-Redesign?type=design&node-id=3050-5404&mode=design)
